pkg/pipeline/PipelineBuilder.go

```diff
@@ -68,6 +68,9 @@ import (
 	"go.uber.org/zap"
 )
 
+const DashboardConfigMap = "dashboard-cm"
+const ConfigMapNamespace = "devtroncd"
```

Comment:
check similar constants already used, remove namespace const.

Reply:
dashboard-cm const is not used, removed namespace const.